extensions/vertx-web/deployment/src/test/java/io/quarkus/vertx/web/SimpleRouteTest.java

```diff
@@ -52,6 +53,12 @@ void hello(RoutingContext context) {
             context.response().setStatusCode(200).end("Hello " + (name != null ? name : "world") + "!");
         }
 
+        @Route(path = "/rx-hello")
+        void rxHello(io.vertx.reactivex.ext.web.RoutingContext context) {
+            String name = context.request().getParam("name");
+            context.response().setStatusCode(200).end("Hello " + (name != null ? name : "world") + "!");
+        }
+
```

Comment:
We probably want a RoutingExchangeImpl test case as well

Reply:
`RoutingExchangeImpl` is already covered by methods accepting `RoutingExchange`.

Reply:
I was thinking that we were missing the negation I mentioned before, and an explicit test would find that missing negation. But agree, another test case would increase test time for not much more coverage